sentry/src/main/java/io/sentry/clientreport/ClientReportRecorderImpl.java

```diff
@@ -0,0 +1,176 @@
+package io.sentry.clientreport;
+
+import io.sentry.DateUtils;
+import io.sentry.SentryEnvelope;
+import io.sentry.SentryEnvelopeItem;
+import io.sentry.SentryItemType;
+import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
+import io.sentry.transport.DataCategory;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class ClientReportRecorderImpl implements ClientReportRecorder {
```

Comment:
```suggestion
public final class ClientReportRecorder implements ClientReportRecorder {
```

We don't use `Impl` anywhere else I guess.

Reply:
`IClientReportRecorder` for the interface then?

Reply:
- [ ] rename `ClientReportRecorderImpl` to `ClientReportRecorder` and `ClientReportRecorder` to `I ClientReportRecorder`